smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftSymbolProvider.kt

```diff
@@ -205,7 +211,7 @@ class SwiftSymbolProvider(private val model: Model, val swiftSettings: SwiftSett
                 .putProperty(SymbolProperty.NESTED_SYMBOL, symbol)
                 .build()
         }
-        return symbol
+        return handleDefaultValue(shape, symbol.toBuilder()).build()
```

Comment:
Before returning the built symbol for member shape, handle any default value it should have if applicable.